railties/lib/rails/generators/rails/app/templates/app/views/layouts/application.html.erb.tt

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html>
+<html lang="<%%= locale %>">
```

Comment:
The `locale` method appears to be undocumented - I assume it's the delegation to the lookup context in `ActionView::Base`. It's probably okay but don't want to accidentally commit us to supporting this method.

Reply:
`I18n.default_locale` seems like a valid choice but if we're interpolating that into the template on all newly generated apps I'd expect it to be first class. Similar to how you can do `t` instead of `I18n.t` in a view, can we make `default_locale` accessible in the view and use that?

Reply:
Thinking about this a little more, does `locale` make more sense than `default_locale`? If the user changes  the locale (like [this](http://guides.rubyonrails.org/i18n.html#managing-the-locale-across-requests)) `default_locale` will still return `en`, but `locale` change along with the user.